lib/Doctrine/DBAL/Types/Type.php

```diff
@@ -182,11 +191,7 @@ public static function getType($name)
      */
     public static function addType($name, $className)
```

Comment:
Hmm, should these `$name` and `$className` also have types defined, because later on eg. that `register` method is expecting certain types.

Reply:
That will not be possible in master due to BC - phpDoc types are sufficient here.